source/guide_filebrowser.rst

```diff
@@ -0,0 +1,121 @@
+.. highlight:: console
+
+.. author:: on4r <https://github.com/on4r>
+
+.. categorize your guide! refer to the current list of tags: https://lab.uberspace.de/tags
+.. tag:: web
+.. tag:: file-storage
+.. tag:: self-hosting
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/filebrowser.png
+      :align: center
+
+############
+File Browser
+############
+
+.. tag_list::
+
+`File Browser`_ provides a file managing interface within a specified directory and it can be used to upload, delete, preview, rename and edit your files.
+It allows the creation of multiple users and each user can have its own directory. It can be used as a standalone app or as a middleware.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`web backends <web-backends>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+License
+=======
+
+File Browser is licensed under the `Apache License 2.0`_
+
+Installation
+============
+
+Download and extract the `latest release`_
+------------------------------------------
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ cd ~/bin
+ [isabell@stardust bin]$ curl --location <url_of_linux_amd64_filebrowser> | tar -xzf - filebrowser
```

Comment:
say exactly which url should be used here, typically the url stays the same and one only needs to replace the version number

Reply:
It looks like we can now easily link to the latest release assets, see: [Linking to releases](https://docs.github.com/en/repositories/releasing-projects-on-github/linking-to-releases) (at the end of the site)